### DISCUSSION TITLE
error message when importing jax

### DISCUSSION THREAD

**Discussion by mingjun-sun** [jax-ml/jax#15402] [Q&A]
This is the result when I want to "import jax". How do I solve this issue?
<img width="879" alt="image" src="https://user-images.githubusercontent.com/50226927/229986796-2b7bf75f-c857-4d46-b62e-bcffb81722ec.png">

This is the result when I run "pip show jax jaxlib".
<img width="464" alt="image" src="https://user-images.githubusercontent.com/50226927/229986868-6715aeda-a198-4f1d-b8a6-009a0b0a3ebf.png">

**Comment by hawkinsp** [comment]
Hmm. This doesn't happen for us, and for me to fix it I'd need to reproduce it.

Is it possible you have a directory named `jax` in your working directory?

**Reply by mingjun-sun** [reply]
I don't have a directory named jax in my working directory. How can I reproduce the problem for you?

**Comment by jakevdp** [comment]
This kind of error can result from importing an old version of `jax`, then installing a new version of jax and attempting to import it in the same runtime. If this is the problem, the fix would be to restart your Python runtime after installing the new version.
